src/index.html

```diff
@@ -12,9 +12,9 @@
         </div>
         <div class="dcs-full-page__main">
           <div class="dcs-full-page__details">
-            <div class="dcs-full-page__logo">
+            <a class="dcs-full-page__logo" href="#">
```

Comment:
Why make this a <div> instead? I'm a nervous that it could affect the styles if they are selecting the <a> tag...

Reply:
Change it to a link so that the cursor changes to a hand when hovering (consistent with other hyperlinks etc.)

Reply:
Will adjust the style to preserve existing size of logo